dcpy/utils/s3.py

```diff
@@ -212,7 +212,7 @@ def upload_folder(
     max_files: int = MAX_FILE_COUNT,
     contents_only: bool = False,
     metadata: dict[str, Any] | None = None,
-    delete_existing: bool = True,
+    keep_existing: bool = False,
```

Comment:
Especially with cli and typer flag options, seemed cleaner to have a variable default to false